src/features/embalm/stepContent/utils/errors.ts

```diff
@@ -0,0 +1,46 @@
+import { Archaeologist } from '../../../../types';
+import { CreateSarcophagusStage } from './createSarcophagus';
+import { formatContractCallException } from '../../../../lib/utils/contract-error-handler';
+
+const processArchCommsException = (offendingArchs: Archaeologist[]) => {
+  if (!!offendingArchs.length) {
+    console.log(
```

Comment:
Do we need a TODO here, since this is merely a log and we might wanna add more logic? I don't even remember what I was thinking here -- I only remember this function used to be beefier and I trimmed it down to this log.

Depending on the direction the eager-dial PR takes, and the [thoughts I left on there](https://github.com/sarcophagus-org/sarcophagus-v2-app/pull/165#discussion_r1015424533), this function might also end up being tossed.

Reply:
Right now I think this is useful to keep for development purposes, but there's a chance we may not need it if the exceptions are being set in state and handled in the UX correctly.